.nvmrc

```diff
@@ -0,0 +1 @@
+14
```

Comment:
Why do we need this?

Reply:
I thought that nvm would automatically load the version declared in this file in the local development environment, but reading the documentation you must run nvm use manually for this. I think it's useful to have it, it doesn't hurt. https://github.com/polkadot-js/api/blob/master/.nvmrc , was just trying to follow polkadot/api structure.

Reply:
I don't think I ever had nvm installed even. Our CI doens't use it either in any of the projects.

Let's remove it until it is actually needed and just specify Node.js version in `package.json`, ideally, `16` from the beginning, it is stable and will become LTS very soon.

Reply:
Removed at : 33564f7